### ISSUE TITLE
Mojang session-server issue

### ISSUE THREAD

**Issue by jammet** [cuberite/cuberite#2870]
Sometimes (rare) this happens:

Trying to connect to Cuberite. I can connect to all other servers just fine. I can resolve other hostnames on that same computer outside of Cuberite, but Cuberite can't do it itself. 
Is this really a networking problem?

```
[12:29:15] LibEvent: Nameserver 127.0.0.1:53 has failed: request timed out.
[12:29:15] LibEvent: All nameservers have failed
[12:29:20] Cannot connect to sessionserver.mojang.com: "Resource temporarily unavailable"
[12:29:20] SecureRequest: Can't connect to sessionserver.mojang.com:
[12:29:20] Kicking player Jammet for "Failed to authenticate account!"
[12:29:48] Cannot connect to sessionserver.mojang.com: "Resource temporarily unavailable"
[12:29:48] SecureRequest: Can't connect to sessionserver.mojang.com:
[12:29:48] Kicking player Jammet for "Failed to authenticate account!"
[12:30:37] Cannot connect to sessionserver.mojang.com: "Resource temporarily unavailable"
[12:30:37] SecureRequest: Can't connect to sessionserver.mojang.com:
[12:30:37] Kicking player Jammet for "Failed to authenticate account!"
[12:32:17] Cannot connect to sessionserver.mojang.com: "Resource temporarily unavailable"
[12:32:17] SecureRequest: Can't connect to sessionserver.mojang.com:
[12:32:41] Cannot connect to sessionserver.mojang.com: "Resource temporarily unavailable"
[12:32:41] SecureRequest: Can't connect to sessionserver.mojang.com:
[12:32:41] Kicking player Jammet for "Failed to authenticate account!"
```

**Comment by Seadragon91** [comment]
Updated your comment, `[code]`, `[/code]` doesn't work here :)

I see this messages under windows, but not under linux.

**Comment by madmaxoft** [comment]
I've seen this several times as well, no idea why it happens, it usually starts working after a while. I suspect something in LibEvent is causing this.

**Comment by peterbell10** [comment]
Was this fixed by #3766 which stopped using LibEvent DNS?
#2418 might be fixed as well.

**Comment by bearbin** [comment]
Since this is suspected fixed, and not updated for over a year, I will close it.
